(design): redesigned login screen with premium social UI

adityashirsatrao007 opened the pull request:
Issue : #4

This PR redesigns the Login screen to be consistent with the new onboarding flow.

- Implemented premium UI with social login options (Google, GitHub).
- Added Guest Access action.
- Ensured consistent branding and color palette (moving away from basic green).
- Integrated official brand icons from Ionicons.

Verified locally on dev server.

07CalC commented:
@adityashirsatrao007, the pr is not upto the requirements
- there needs to be only one way of looging in, which is via google, we do not support any other way 
- there is opencode mention, i have no idea why
- make this screen's color scheme to match that of the getting started screen
- you should remove the `login` header with the back button, instead implement it manually

adityashirsatrao007 commented:
> @adityashirsatrao007, the pr is not upto the requirements
> 
> * there needs to be only one way of looging in, which is via google, we do not support any other way
> * there is opencode mention, i have no idea why
> * make this screen's color scheme to match that of the getting started screen
> * you should remove the `login` header with the back button, instead implement it manually

yo I'll do that

adityashirsatrao007 commented:
Hey @shaurpro, updated the login screen as requested! 🦊✨

- Switched the theme to the official **orange/magenta** business palette.
- Removed the default header and implemented a **custom back button** for a cohesive look.
- Integrated the **brand logo** to match the revamped 'Getting Started' screen.
- Cleaned up the footer for a more minimalist aesthetic.

Ready for another look!

adityashirsatrao007 commented:
> Hey @07CalC , updated the login screen as requested! 🦊✨
> 
> * Switched the theme to the official **orange/magenta** business palette.
> * Removed the default header and implemented a **custom back button** for a cohesive look.
> * Integrated the **brand logo** to match the revamped 'Getting Started' screen.
> * Cleaned up the footer for a more minimalist aesthetic.
> 
> Ready for another look!

07CalC commented:
@adityashirsatrao007 i see no changes except for the header, you'd rather want to work on something else than ui, i'll open some new issues check if they interests you

adityashirsatrao007 commented:
> @adityashirsatrao007 i see no changes except for the header, you'd rather want to work on something else than ui, i'll open some new issues check if they interests you


https://github.com/user-attachments/assets/2c98d437-0126-4de5-a177-0148765ff48b
